guides/source/active_job_basics.md

```diff
@@ -269,6 +269,12 @@ UserMailer.welcome(@user).deliver_now
 UserMailer.welcome(@user).deliver_later
 ```
 
+You can also configure `ActionMailer::DeliveryJob` to use a specific queue
+adapter, distinct from the one set on `ActiveJob::Base`:
+
+```ruby
+Rails.application.config.action_mailer.delivery_job.queue_adapter = :sneakers
```

Comment:
The two-level name, and user-visible mention of `delivery_job`, seems unfortunate. My first inclination is to call it `action_mailer.delivery_queue` or similar... but maybe we do have other configurable parameters on the job, that people would want to set?